<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="19682" started-at="2018-07-25T07:09:20Z" finished-at="2018-07-25T07:09:39Z">
    <groups>
    </groups>
    <test name="Test1" duration-ms="640" started-at="2018-07-25T07:09:20Z" finished-at="2018-07-25T07:09:20Z">
      <class name="Framworktests.Logintest">
        <test-method status="FAIL" signature="LogInvalid()[pri:0, instance:Framworktests.Logintest@3ac3fd8b]" name="LogInvalid" duration-ms="20" started-at="2018-07-25T12:39:20Z" finished-at="2018-07-25T12:39:20Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Tested] but found [Tested123]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Tested] but found [Tested123]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at Framworktests.Logintest.LogInvalid(Logintest.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LogInvalid -->
        <test-method status="PASS" signature="LoginValid()[pri:0, instance:Framworktests.Logintest@3ac3fd8b]" name="LoginValid" duration-ms="0" started-at="2018-07-25T12:39:20Z" finished-at="2018-07-25T12:39:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginValid -->
      </class> <!-- Framworktests.Logintest -->
    </test> <!-- Test1 -->
    <test name="Test2" duration-ms="18325" started-at="2018-07-25T07:09:21Z" finished-at="2018-07-25T07:09:39Z">
      <class name="Framworktests.Registrationtest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:Framworktests.Registrationtest@3fb6a447]" name="beforeTest" is-config="true" duration-ms="11688" started-at="2018-07-25T12:39:21Z" finished-at="2018-07-25T12:39:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="f()[pri:0, instance:Framworktests.Registrationtest@3fb6a447]" name="f" duration-ms="6627" started-at="2018-07-25T12:39:33Z" finished-at="2018-07-25T12:39:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:Framworktests.Registrationtest@3fb6a447]" name="afterTest" is-config="true" duration-ms="1" started-at="2018-07-25T12:39:39Z" finished-at="2018-07-25T12:39:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- Framworktests.Registrationtest -->
      <class name="Framworktests.Logintest">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:Framworktests.Logintest@3ac3fd8b]" name="beforeSuite" is-config="true" duration-ms="30" started-at="2018-07-25T12:39:20Z" finished-at="2018-07-25T12:39:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:Framworktests.Logintest@3ac3fd8b]" name="afterSuite" is-config="true" duration-ms="0" started-at="2018-07-25T12:39:39Z" finished-at="2018-07-25T12:39:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- Framworktests.Logintest -->
    </test> <!-- Test2 -->
  </suite> <!-- Suite -->
</testng-results>
